test/CDB_DateToNumberTest.sql

```diff
@@ -0,0 +1,2 @@
+SELECT * FROM CDB_DateToNumber('1999-01-08 00:00:00+00'::timestamp);
```

Comment:
semantically, the `+00` is not natural for a timestamp, so if we want to check specifically for non-timezone I'd remove it